spec/controllers/coronavirus_form/business_details_controller_spec.rb

```diff
@@ -83,8 +83,7 @@
         company_location: I18n.t("coronavirus_form.questions.business_details.company_location.options.european_union.label"),
       ).except(:company_postcode)
 
-      expect(response).to have_http_status(:unprocessable_entity)
-      expect(response).to render_template(current_template)
+      expect(response).to redirect_to(contact_details_path)
```

Comment:
This merge commit seems to have removed the `expect(response).to have_http_status(:unprocessable_entity)` that you made in a previous commit

Reply:
Hi Bruce, that was intentional. I thought if we were expecting a redirect then we should not be expecting a 422 error. At least i think that's what it's supposed to be doing 🤔